tutor/src/tours/add-homework.json

```diff
@@ -26,7 +26,7 @@
   "steps": [
     {
       "title": "First, select a chapter",
-      "body": "Select a chapter you want to assign, then scroll down and click “show problems.” Uncheck any sections you don’t want to cover."
+      "body": "Select a chapter you want to assign, then scroll down and click \"Show Problems.\" Uncheck any sections you don't want to cover."
     }
```

Comment:
hmmm...the word doc has the UTF curly quotes but the mockups don't.  I think they are the default in the word doc, and that's the only reason why